tts-voicerss/src/main/java/org/mobicents/media/server/tts/voicerss/VoiceRssEngine.java

```diff
@@ -0,0 +1,193 @@
+						}
+						is.close();
+					} catch (IOException e) {
+						// TODO Auto-generated catch block
+						e.printStackTrace();
+					}
+				}
+			} else {
+				if ( logger.isInfoEnabled() ) {
+					logger.info("VoiceRssEngine error, status code: " + line.getStatusCode() + (" reason phrase: ") + line.getReasonPhrase());
+				} else {
+					// Do nothing
+				}
+				final StringBuilder buffer = new StringBuilder();
+				buffer.append(line.getStatusCode()).append(" ").append(line.getReasonPhrase());
+				throw new SpeechSynthesizerException(buffer.toString());
+			}
+		} catch (UnsupportedEncodingException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+		return speakData;
+	}
+	
+	private void load(final Configuration configuration) throws RuntimeException {
+		// Initialize
+		myMapLanguage.put("ca", configuration.getString("languages.catalan"));
+		myMapLanguage.put("zh-cn", configuration.getString("languages.chinese-china"));
+		myMapLanguage.put("zh-hk", configuration.getString("languages.chinese-hongkong"));
+		myMapLanguage.put("zh-tw", configuration.getString("languages.chinese-taiwan"));
+		myMapLanguage.put("da", configuration.getString("languages.danish"));
+		myMapLanguage.put("nl", configuration.getString("languages.dutch"));
+		myMapLanguage.put("en-au", configuration.getString("languages.english-australia"));
+		myMapLanguage.put("en-ca", configuration.getString("languages.english-canada"));
+		myMapLanguage.put("en-gb", configuration.getString("languages.english-greatbritain"));
+		myMapLanguage.put("en-in", configuration.getString("languages.english-india"));
+		myMapLanguage.put("en", configuration.getString("languages.english-us"));
+		myMapLanguage.put("fi", configuration.getString("languages.finish"));
+		myMapLanguage.put("fr-ca", configuration.getString("languages.french-canada"));
+		myMapLanguage.put("fr", configuration.getString("languages.french-france"));
+		myMapLanguage.put("de", configuration.getString("languages.german"));
+		myMapLanguage.put("it", configuration.getString("languages.italina"));
+		myMapLanguage.put("ja", configuration.getString("languages.japanese"));
+		myMapLanguage.put("ko", configuration.getString("languages.korean"));
+		myMapLanguage.put("nb", configuration.getString("languages.norwegian"));
+		myMapLanguage.put("pl", configuration.getString("languages.polish"));
+		myMapLanguage.put("pt-br", configuration.getString("languages.portuguese-brasil"));
+		myMapLanguage.put("pt", configuration.getString("languages.portuguese-portugal"));
+		myMapLanguage.put("ru", configuration.getString("languages.russian"));
+		myMapLanguage.put("es-mx", configuration.getString("languages.spanish-mexico"));
+		myMapLanguage.put("es", configuration.getString("languages.spanish-spain"));
+		myMapLanguage.put("sv", configuration.getString("languages.swedish"));
+	}
+
+	public String getAPIkey() {
```

Comment:
I don't think we need a getter for API key. What other entity needs access to this information?